frontend/packages/ocs-independent-plugin/src/queries.ts

```diff
@@ -0,0 +1,9 @@
+export enum IndependentDashboardQuery {
```

Comment:
Assuming OCS/independent plugin depends on Ceph plugin, both this `enum` and the `const` object below can be reused from `packages/ceph-storage-plugin/src/constants/queries.ts`.